android/app/src/main/java/io/peggyjo/peggymeter/UpdateMoodActivity.java

```diff
@@ -12,7 +13,7 @@
 import io.peggyjo.peggymeter.database.DataController;
 import io.peggyjo.peggymeter.model.LogEntry;
 
-public class UpdateMoodActivity extends AppCompatActivity {
+public class UpdateMoodActivity extends AppCompatActivity implements MediaPlayer.OnCompletionListener {
```

Comment:
Please move the implements decl to the next line.

Reply:
Right.